iconic-tpd.php

```diff
@@ -9,6 +9,12 @@
  * Text Domain: iconic-tdp
  */
 
+namespace Iconic\Temporarily_Disable_Plugins;
+
+use function add_action;
+use function add_filter;
+// TODO Import all WordPress function.
```

Comment:
*s

Reply:
Should we import the rest - is it required?

Reply:
No it is not.
In a sense that new lines are not required in a PHP file.

Reply:
Custom namespaced functions may have the very same name as a WP core function.

Reply:
I suppose it isn't an issue because we don't have any functions with the same name, right?

Reply:
It is not, you're right.